.github/workflows/library.yml

```diff
@@ -77,6 +73,8 @@ jobs:
             task: 'slow'
           - os: 'windows'
             task: 'pyright'
+          - os: 'windows'
```

Comment:
no propagate for ubuntu?

Reply:
Actually that's the "ignore" list :)... that essentially means only Ubuntu.